public/omni/getting-started/getting-started.mdx

```diff
@@ -18,122 +18,151 @@ If you would like to watch a quick video of the process you can follow along her
-* proxying of workload HTTP access
-* simple scaling up and down of clusters
-* the concept of Machine Sets, that let you manage your infrastructure by classes
+    ![Worker node button](./images/getting-started-worker-node.png)
 
-If you want to declaratively manage your clusters and infrastructure declaratively, as code, check out [Cluster Templates](../reference/cluster-templates.md).
+6. Click **Create Cluster**.
 
-### Destroy the Cluster
+You’ll be redirected to the **Cluster Overview** page, where you can monitor the cluster’s creation progress.
 
-When you are all done, you can remove the cluster by clicking "Destroy Cluster", in the bottom right of the Cluster Overview panel. This will wipe the machines and return them to the Available state.
+## Step 4: Download the `kubeconfig` file
 
-## Cluster example
+Click the **⋯** button next to your cluster, then select **Download kubeconfig** to download the file.
 
-We have an example of a managed cluster complete with a monitoring stack and application management. It can be found in our [community contrib repo](https://github.com/siderolabs/contrib/blob/main/examples/omni).
+![Download kubeconfig](./images/getting-started-download-kubeconfig.png)
 
-### Components
+## Step 5: Set default configuration paths
 
-The contrib example includes:
+To make `omnictl`, `talosctl`, and `kubectl` automatically detect your cluster configuration, move your downloaded files to their default locations:
 
-* [Cilium](https://cilium.io/get-started/) for cluster networking
-* [Hubble](https://docs.cilium.io/en/stable/gettingstarted/hubble\_intro/) for network observability
-* [ArgoCD](https://argo-cd.readthedocs.io/en/stable/) for application management
-* [Rook/Ceph](https://rook.io/docs/rook/latest-release/Getting-Started/intro/) for persistent storage
-* [Prometheus](https://prometheus.io/docs/introduction/overview/) for metrics collection and alerting
-* [Grafana](https://grafana.com/docs/grafana/latest/introduction/) for metrics visualization
+1. Create the required default directories if they do not exist:
 
-### Use
+```bash
+mkdir -p ~/.config/omni/config
+mkdir -p ~/.talos
+mkdir -p ~/.kube
+```
 
-You will need to copy the contents of the `omni` directory to a git repository that can be accessed by the cluster you create. Update the [ArgoCD ApplicationSet](https://github.com/siderolabs/contrib/blob/main/examples/omni/apps/argocd/argocd/bootstrap-app-set.yaml) template to reference your new git repo, and regenerate the ArgoCD bootstrap patch.
+2. Move the downloaded configuration files to their default paths:
 
 ```bash
-sed -i 's|https://github.com/siderolabs/contrib.git|<your-git-repo>|' apps/argocd/argocd/bootstrap-app-set.yaml
-kustomize build apps/argocd/argocd | yq -i 'with(.cluster.inlineManifests.[] | select(.name=="argocd"); .contents=load_str("/dev/stdin"))' infra/patches/argocd.yaml
+mv <path-to-your-omniconfig-file> ~/.config/omni/config/config
```

Comment:
I would make these `$HOME/Downloads/omniconfig $HOME/.config/omni/config` because that's most likely where the file will be (linux and mac) and if they're on windows the `mv` command wouldn't work anyway :sweat_smile: